### PR TITLE
chore: update testing infra

### PR DESCRIPTION
**Pull request by jaredperreault-okta**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [guidelines](/okta/okta-angular/blob/master/CONTRIBUTING.md#commit)
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Adding Tests
- [ ] Build related changes
- [x] CI related changes
- [ ] Documentation changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: OKTA-534638

## Does this PR introduce a breaking change?
- [ ] Yes
- [x] No

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 node_modules
 dist
-test-reports
\ No newline at end of file
+test-reports
+test/apps/angular-v*
```
Comment (denysoblohin-okta) [inline .eslintignore:4]: Why ignore test apps from eslint?

Comment (jaredperreault-okta) [reply .eslintignore:4]: The linter was throwing errors which were being ignored. The test apps do not contain a copy of `@okta/angular` during build time, so the linter flags it

```bash
Start lint
$ eslint --ext .js,.ts  .

/root/okta/okta-angular/test/apps/angular-v12/src/app/app.component.spec.ts
  7:8  error  Unable to resolve path to module '@okta/okta-angular'  import/no-unresolved

/root/okta/okta-angular/test/apps/angular-v12/src/app/app.component.ts
  15:49  error  Unable to resolve path to module '@okta/okta-angular'  import/no-unresolved

/root/okta/okta-angular/test/apps/angular-v12/src/app/app.module.ts
  26:8  error  Unable to resolve path to module '@okta/okta-angular'  import/no-unresolved

/root/okta/okta-angular/test/apps/angular-v12/src/app/protected.component.ts
  14:27  error  Unable to resolve path to module '@okta/okta-angular'  import/no-unresolved

/root/okta/okta-angular/test/apps/angular-v12/src/app/public.component.ts
  14:27  error  Unable to resolve path to module '@okta/okta-angular'  import/no-unresolved

/root/okta/okta-angular/test/apps/angular-v12/src/app/sessionToken-login.component.ts
  14:27  error  Unable to resolve path to module '@okta/okta-angular'  import/no-unresolved

/root/okta/okta-angular/test/apps/angular-v13/src/app/app.component.spec.ts
  7:8  error  Unable to resolve path to module '@okta/okta-angular'  import/no-unresolved

/root/okta/okta-angular/test/apps/angular-v13/src/app/app.component.ts
  15:49  error  Unable to resolve path to module '@okta/okta-angular'  import/no-unresolved
```

Comment (shuowu-okta) [reply .eslintignore:4]: Possible to ignore this specific rule in test apps eslintrc other than ignore all lint checks?

Comment (shuowu-okta) [reply .eslintignore:4]: looks like there is a setting to ignore specific packages
https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-unresolved.md#ignore

**Review by shuowu-okta — APPROVED** [review]

File: test/apps/angular-v12/src/app/app.component.ts [hunk 2]
```diff
@@ -10,6 +10,7 @@
  * See the License for the specific language governing permissions and limitations under the License.
  */
 
+/*eslint import/no-unresolved: [2, { ignore: ['@okta/okta-angular$'] }]*/
```
Comment (shuowu-okta) [inline test/apps/angular-v12/src/app/app.component.ts:13]: nit: maybe just another .eslintrc under test/apps, which should be able to inherit rules from the root

**Pull request merged**